nlmod/gwf/recharge.py

```diff
@@ -163,15 +200,308 @@ def ds_to_evt(gwf, ds, pname="evt", nseg=1, surface=None, depth=None, **kwargs):
+        The applied infiltration rate of the UZF cell (:math:`LT^{-1}`). When passed as
+        string, the array is obtained from ds. The default is "recharge".
+    pet : float, str or array-like
+        The potential evapotranspiration rate of the UZF cell and specified GWF cell.
+        Evapotranspiration is first removed from the unsaturated zone and any remaining
+        potential evapotranspiration is applied to the saturated zone. only
+        used if SIMULATE_ET. When passed as string, the array is obtained from ds. The
+        default is "evaporation".
+    extdp : float, optional
+        Value that defines the evapotranspiration extinction depth of the UZF cell, in
+        meters below the top of the model. Set to 2.0 meter when None. The default is
+        None.
+    extwc : float, optional
+        The evapotranspiration extinction water content of the UZF cell. Only used if
+        SIMULATE_ET and UNSAT_ETWC. Set to thtr when None. The default is None.
+    ha : float, optional
+        The air entry potential (head) of the UZF cell. Only used if SIMULATE_ET and
+        UNSAT_ETAE. Set to 0.0 when None. The default is None.
+    hroot : float, optional
+        The root potential (head) of the UZF cell. Only used if SIMULATE_ET and
+        UNSAT_ETAE. Set to 0.0 when None. The default is None.
+    rootact : float, optional
+        the root activity function of the UZF cell. ROOTACT is the length of roots in
+        a given volume of soil divided by that volume. Values range from 0 to about 3
+        :math:`cm^{-2}`, depending on the plant community and its stage of development.
+        Only used if SIMULATE_ET and UNSAT_ETAE. Set to 0.0 when None. The default is
+        None.
+    simulate_et : bool, optional
+        If True, ET in the unsaturated (UZF) and saturated zones (GWF) will be
+        simulated. The default is True.
+    linear_gwet : bool, optional
+        If True, groundwater ET will be simulated using the original ET formulation of
+        MODFLOW-2005. When False, and square_gwet is True as an extra argument, no
+        evaporation from the saturated zone will be simulated. The default is True.
+    unsat_etwc : bool, optional
+        If True, ET in the unsaturated zone will be simulated as a function of the
+        specified PET rate while the water content (THETA) is greater than the ET
+        extinction water content (EXTWC). The default is False.
+    unsat_etae : bool, optional
+        If True, ET in the unsaturated zone will be simulated using a capillary pressure
+        based formulation. Capillary pressure is calculated using the Brooks-Corey
+        retention function. The default is False.
+    obs_depth_interval : float, optional
+        The depths at which observations of the water depth in each cell are added. The
+        user-specified depth must be greater than or equal to zero and less than the
+        thickness of GWF cellid (TOP - BOT).
+        The
```

Comment:
The....

Reply:
Maybe the intention, but we should explain that this creates a CSV output file with water contents for each UZF cell?

Reply:
`obs_z` is missing from the docstring